tools/web3js-example/scripts/contractCall.js

```diff
@@ -20,14 +20,14 @@
 
 require('dotenv').config();
 const fs = require('fs');
-const Web3 = require('web3');
-const Web3HttpProvider = require('web3-providers-http');
+const { Web3 } = require('web3');
 
 module.exports = async (address, msg) => {
   const { abi } = await JSON.parse(fs.readFileSync(__dirname + '/../contract/Greeter.json'));
-  const web3 = new Web3(new Web3HttpProvider(process.env.RELAY_ENDPOINT));
+  const httpProvider = new Web3.providers.HttpProvider(process.env.RELAY_ENDPOINT);
+  const web3 = new Web3(httpProvider);
```

Comment:
Q: why the breakout if `httpProvider` isn't used elsewhere?

Reply:
It's not needed.  Updated.